### PR TITLE
Additional methods for the Matrix

### PR DESCRIPTION
**Pull request by nkrusch**
This changeset includes what was listed in issue #12.

### CODE REVIEW

File: pymwp/matrix.py [hunk 1]
```diff
@@ -172,3 +189,80 @@ def resize(matrix: List[List[Polynomial]], new_size: int) -> List[List[Polynomia
...
+    # element-wise comparison
+    for row_index, column in enumerate(matrix1):
+        for col_index, value in enumerate(column):
+            if matrix2[row_index][col_index] != value:
+                return False
+
+    return True
+
+
+def fixpoint(matrix: List[List[Any]]) -> List[List[Any]]:
```
Comment (aubertc) [inline pymwp/matrix.py:247]: So I guess now the question may be "Do we need both this method _and_  [def fixpoint(self) -> Relation](https://github.com/seiller/pymwp/blob/4fe2d96d48d879f1d1e085583dab1dd0447d2e65/pymwp/relation.py#L208) ?

File: tests/test_matrix.py [hunk 2]
```diff
@@ -112,3 +114,64 @@ def test_decode():
...
+
+
+def test_matrix_size_not_equals():
+    m1 = [[o, o, o], [o, o, o], [o, o, o]]
+    m2 = [[o, o], [o, o]]
+
+    assert equals(m1, m2) is False
+
+
+def test_fixpoint():
```
Comment (aubertc) [inline tests/test_matrix.py:150]: Great. I can try to come up with more complex examples at some point.

**Comment by aubertc** [comment]
We can merge or wait for https://github.com/seiller/pymwp/tree/simpl to merge first, either way is fine with me.

File: pymwp/matrix.py (hunk 1 above)
Comment (nkrusch) [reply pymwp/matrix.py:247]: Fixpoint in relation does homogenization also. This function assumes you have two matrices of same size. I would say, they have different use cases.

File: tests/test_matrix.py (hunk 2 above)
Comment (nkrusch) [reply tests/test_matrix.py:150]: Yes, I would like to have a more advanced case to test. If you spot one that changes when doing things by hand make a note of it for me please.

**Comment by aubertc** [comment]
I need to perform fixpoint calculation with simpler polynomial: merging now. I hope it's ok with you all.

**Pull request merged**